nuxeo-features/nuxeo-thumbnail/src/main/java/org/nuxeo/ecm/platform/thumbnail/operation/RecomputeThumbnails.java

```diff
@@ -41,16 +42,17 @@
     public static final String DEFAULT_QUERY = "SELECT * FROM Document WHERE ecm:mixinType = 'Thumbnail' AND thumb:thumbnail/data IS NULL AND ecm:isVersion = 0 AND ecm:isProxy = 0 AND ecm:isTrashed = 0";
 
     @Context
-    protected CoreSession session;
+    protected OperationContext ctx;
```

Comment:
The operation should not be touched by this commit

Reply:
This constant is updating the operation